src/main/scala/util/package.scala

```diff
@@ -234,23 +234,55 @@ package object util {
     map.view.map({ case (k, vs) => k -> vs.toList }).toList
   }
 
+/** provides operators useful for working with bidirectional [[Bundle]]s
+  * 
+  * In terms of [[Flipped]] with a producer 'p' and 'consumer' c:
+  * c :<= p // means drive all unflipped fields of 'c' from 'p' (e.g.: valid/bits)  
+  * c :=> p // means drive all flipped fields of 'c' from 'p' (e.g.: ready)  
+  * c :<> p // do both of the above
+  * p :<> c // do both of the above, but you'll probably get a Gender error later.
```

Comment:
`Gender` has been replaced by `Flow`. Also if this results in flow errors in FIRRTL, I think it would be better to just catch them here instead.

Reply:
You propose to try and introspect the actual direction? And if it cannot be determined (ie: is a wire), just trust the user? That is fine by me.

Reply:
@jackkoenig  do you have any plan of supporting these operators in chisel?